projects/bp-gallery/src/shared/style.module.scss

```diff
@@ -0,0 +1,41 @@
+// color scheme
+
+$primaryColor: #7e241d;
+$secondaryColor: #404173;
+$backgroundColor: #f5f5f5;
+$textColor: #0a0a0a;
+$decadeColor: #690e6e;
+$descriptionColor: #7e241d;
+$personColor: #404173;
+$locationColor: #2c6e49;
+$keywordColor: #d18c2d;
+$markAsTextColor: #b95e3d;
+$linkColor: #b64b5a;
+$collectionColor: #a053a7;
+$archiveColor: #446ab7;
+$titleColor: #5f5e5e;
+$allSearchColor: #84b0c4;
+$likeColor: rgb(96 165 250); // blue-400 from tailwind
```

Comment:
can we change this to #60a5fa ? I think it's better if the colours are all in Hex :)